proxy/core/connection/connection.py

```diff
@@ -66,6 +67,7 @@ def recv(
             (len(data), self.tag),
         )
         # logger.info(data)
+        logger.info('recv from %s, data = |%s|', self.tag, data)
```

Comment:
same here

Reply:
Can remove too

Reply:
Same here, we should not keep this line.  Comment this out.